test/tests/fvkernels/mms/skewness-correction/two_term_extrapol/tests

```diff
@@ -0,0 +1,26 @@
+[Tests]
+  design = 'fv_design.md'
+  issues = '#16239'
+  [average]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAverageStencil
+    requirement = 'The system shall display first order convergence when using an average-based face-interpolation together with extrapolated BCs.'
+    method = '!dbg'
+    min_parallel = 1
+    ad_indexing_type = 'global'
+    required_python_packages = 'pandas matplotlib'
+    max_time = 100000000
```

Comment:
how long is it?
maybe it needs to go in heavy

Reply:
About 2 secs, definitely not the heaviest test. I removed `max_time`.